src/Core/routing/navbar.js

```diff
@@ -43,7 +43,7 @@ export default _.compose([connect(stateToProps), withTranslation(), makeExtendab
                                 <NavLink className="nav-link" id={route.menuName} to={route.path}>{t(route.menuName)}</NavLink>
                             </li>
                             : route.TellMe ?
-                                <div className='d-lg-none' style={{color: Colors.blackOpacity05}}>
+                                <div className='d-lg-none' key={i} style={{color: Colors.blackOpacity05}}>
```

Comment:
die variable i sagt mir nichts. bitte gute benennung finden. nur in zukunft - jetzt ist es ok .